File: datalib/src/main/java/nl/acidcats/tumblrlikes/data_impl/photodata/gateway_impl/greendao/database/migration/Migration.java
package nl.acidcats.tumblrlikes.data_impl.photodata.gateway_impl.greendao.database.migration;

import org.greenrobot.greendao.database.Database;

/**
 * Created by stephan on 11/05/2017.
 */
public interface Migration {
    Integer getVersion();

    void runMigration(Database db);
}

File: datalib/src/main/java/nl/acidcats/tumblrlikes/data_impl/photodata/gateway/greendao/database/migration/Migration.kt
package nl.acidcats.tumblrlikes.data_impl.photodata.gateway.greendao.database.migration

import org.greenrobot.greendao.database.Database

/**
 * Created by stephan on 11/05/2017.
 */
interface Migration {
    val version: Int

    fun runMigration(db: Database)
}
